planner/core/cbo_test.go

```diff
@@ -367,7 +367,7 @@ func (s *testAnalyzeSuite) TestIndexRead(c *C) {
 		},
 		{
 			sql:  "select sum(a) from t1 use index(idx) where a = 3 and b = 100000 group by a limit 1",
-			best: "IndexLookUp(Index(t1.idx)[[3,3]], Table(t1)->Sel([eq(test.t1.b, 100000)]))->StreamAgg->Limit",
+			best: "IndexLookUp(Index(t1.idx)[[3,3]], Table(t1)->Sel([eq(test.t1.b, 100000)]))->Projection->StreamAgg->Limit",
```

Comment:
Why this query would have a `Projection` since `a` is a column?

Reply:
The type of  t1.a is int, and we only have implemented aggregation sum functions for decimal and float64 now;
So we must convert t1.a to float64, which is done in the Projection;